Marlin/SanityCheck.h

```diff
@@ -1146,7 +1146,7 @@ static_assert(1 >= 0
 static_assert(1 >= 0
   #if ENABLED(ULTIMAKERCONTROLLER) \
       && DISABLED(SAV_3DGLCD) && DISABLED(miniVIKI) && DISABLED(VIKI2) \
-      && DISABLED(ELB_FULL_GRAPHIC_CONTROLLER) && DISABLED(PANEL_ONE)
+      && DISABLED(ELB_FULL_GRAPHIC_CONTROLLER) && DISABLED(PANEL_ONE) && DISABLED(MKS_OLED13_128x64_FULL_GRAPHICS_CONTROLLER)
     + 1
   #endif
   #if ENABLED(REPRAP_DISCOUNT_SMART_CONTROLLER) && DISABLED(REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER)
```

Comment:
Instead, since enabling `MKS_OLED13_128x64_FULL_GRAPHICS_CONTROLLER` enables `REPRAP_DISCOUNT_SMART_CONTROLLER`, this is where you would add the clause.

Reply:
OK, I see.  Done